package.json

```diff
@@ -183,6 +183,11 @@
     },
     "setupFiles": [
       "<rootDir>/test/setup.js"
+    ],
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/__fixtures__/",
+      ".history"
```

Comment:
What is this change about?

Reply:
Jest likes to explore all the folders. This normally defaults to `node_modules` but I needed to add `.history` so Jest doesn't run on all the previous versions of my files